fastlane/Fastfile

```diff
@@ -43,4 +69,11 @@ platform :android do
         upload_to_play_store(json_key: ENV["ANDROID_JSON_KEY_FILE"], track: 'production')
   end
 
+  desc "Deploy a production version with auto increment of version code"
+    lane :production_push do
+          sh "fastlane increment_version_code"
+          sh "fastlane production"
```

Comment:
Don't use 'sh' - the other lanes are available to be called as functions I believe (ie. increment_version_code)